### PR TITLE
cbbw toronto

### LINKED ISSUE #13: bwong - checklist
- [x] Make a Branch
- [ ] Work Locally
- [ ] Make a Pull Request
- [ ] Collaborate
- [ ] Merge!

### PR DESCRIPTION
**Pull request by cbbw**
create file with restaurants

Closes #13

### CODE REVIEW

File: cbbw-toronto.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 1. Yutaka on Dundas - sushi (west of bay)
```
Comment (fvalerach) [inline cbbw-toronto.md]: address?

File: cbbw-toronto.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
...
 1. Santouka on Dundas - ramen (east of yonge)
+1. Mo Ramyun at Baldwin - korean fusion
```
Comment (fvalerach) [inline cbbw-toronto.md]: good stuff

**Review by fvalerach — COMMENTED** [review]
halp!

File: cbbw-toronto.md (hunk 1 above)
Comment (cbbw) [reply cbbw-toronto.md]: it's on dundas - west of bay.... read carefully

**Pull request merged**